### PR TITLE
W04 d03 express review

### PR DESCRIPTION
**Pull request by mtvillwock**

### CODE REVIEW

File: w04-associations-and-auth/d3-express-review/views/index.ejs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <div id="page-wrapper" class="container">
+    <header class="page-header">
+      <h1>General Pizza Assembly</h1>
+    </header>
+
+    <!-- include form partial -->
+    <% include ./_create %>
+
+    <h3>Your Pizzas</h3>
+    <% pizzas.forEach(function(pizza) { %>
```
Comment (tgaff) [inline w04-associations-and-auth/d3-express-review/views/index.ejs]: pizzas isn't passed here

File: w04-associations-and-auth/d3-express-review/server.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+var db = require("./models/index");
+
+app.set("view engine", "ejs");
+
+app.use(express.static("public"));
+app.use(bodyParser.urlencoded({extended: true}));
+
+
+app.get('/', function(req, res) {
+  res.render("index");
```
Comment (tgaff) [inline w04-associations-and-auth/d3-express-review/server.js]: @mtvillwock this view looks like it needs `pizzas`

File: w04-associations-and-auth/d3-express-review/views/index.ejs (hunk 1 above)
Comment (mtvillwock) [inline w04-associations-and-auth/d3-express-review/views/index.ejs]: Fixed this

File: w04-associations-and-auth/d3-express-review/server.js (hunk 2 above)
Comment (mtvillwock) [inline w04-associations-and-auth/d3-express-review/server.js]: fixed this

**Comment by tgaff** [comment]
This looked good to me @mtvillwock do you want it merged here or are you just going to link to it in it's other repo?  GTM

**Comment by mtvillwock** [comment]
@tgaff I already posted it as a [new repo](https://github.com/mtvillwock/pizza-app), so I think we can close this PR.

**Pull request closed without merging**